packages/components/src/components/assets/AssetShareIndicator/AssetShareIndicator.tsx

```diff
@@ -1,34 +1,128 @@
-import { SVGProps } from 'react';
+import { Dispatch, SetStateAction, SVGProps, useEffect, useState } from 'react';
 import { motion, AnimationProps, SVGMotionProps } from 'framer-motion';
 import styled, { useTheme } from 'styled-components';
-import { coinsColors } from '@trezor/theme';
+import { coinsColors, spacingsPx } from '@trezor/theme';
 import { motionEasing } from '../../../config/motion';
-import { CoinLogo, CoinLogoProps } from '../CoinLogo/CoinLogo';
-
-const Container = styled.div`
-    position: relative;
-    align-items: center;
-    display: flex;
-    justify-content: center;
-    width: 48px;
-    height: 48px;
-    border-radius: 50%;
-`;
+import { CoinLogo, CoinLogoProps, QUALITY_SIZE, useAssetUrl } from '../CoinLogo/CoinLogo';
+
+const Container = styled.div<{ $size: number }>(
+    ({ $size }) => `
+        position: relative;
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        width: ${$size}px;
+        height: ${$size}px;
```

Comment:
nitpick: maybe it looks better to pass size just here like:

width: ${({ $size }) => $size}px;
height: ${({ $size }) => $size}px;

And not to all styles (applies to all occurrences inside this PR)

Reply:
Yes. There is also another way how to write this code to achieve expected.